.github/workflows/build.yaml

```diff
@@ -228,12 +228,21 @@ jobs:
         run: |
           make integration-test SUPPRESS_TF_OUTPUT=true EXPORT_KIND_LOGS_ROOT=${{ steps.create_kind_export_log_root.outputs.log_root }}
       - name: ENT tests using Kustomize
+        # Ideally we only need to test the latest release of OSS Vault.
+        if: ${{ matrix.vault-version == '1.13.1' }}
```

Comment:
It doesn't look like the VaultDynamicSecret tests run with helm? Maybe we should hold off on this condition until they do?

Reply:
Yeah, I was on the fence about that too.
The reason I went for this approach is that I wanted to avoid having 2 sets of "expansive" tests which don't apply the matrix filter just to keep the runtime down.

Do we expect that VDS test would be somehow more prone to potential issues from combinations of Vault version  and k8s version than the other tests (which already support helm)?

I suppose there isn't a big reason to introduce this change here unless it starts to make tests run too long, thoughts? I'm fine reverting it.

Reply:
Yeah, I dunno, I'm just worried we're losing coverage. Maybe shift the condition to the ENT Helm tests for now, with a todo to switch once the dynamic tests work with helm?

Reply:
Okay took out the conditional so we're running the full matrix against Helm and Kustomize for Vault ENT, and left a TODO comment to put it in in the future.
I think they run in parallel anyways, so shouldnt increase the time right now, I'll keep an eye out though!